src/app/blog/page.tsx

```diff
@@ -64,6 +64,7 @@ const featuredPostSlug = path.parse(featured_entry).name
 const featuredPost = getBlogPostData(featuredPostSlug)
 
 export const metadata = createMetadata({
+  image: { src: featuredPost.image?.src },
   seo,
```

Comment:
I think I would prefer if we add the image field directly to `SeoMetadata` and then use it like so:

```
export const metadata = createMetadata({
  seo: {
    ...seo,
    image: graphicsData.digest.data.src,  // Add image inline to seo
  },
  path: PATHS.DIGEST.path,
})
```